torch/distributed/optim/post_localSGD_optimizer.py

```diff
@@ -1,5 +1,6 @@
 import torch
 import torch.distributed.algorithms.model_averaging.averagers as averagers
+from warnings import warn
```

Comment:
To avoid the CI failure, you can change this to `import warnings` and then call `warnings.warn()` in your code. I am not sure if there is a better way. cc: @rohan-varma

Reply:
Updated, will also take a note for this for future